src/main.py

```diff
@@ -171,13 +173,25 @@ def _run_testing(env: BaseEnv, approach: BaseApproach) -> Metrics:
     metrics["num_total"] = len(test_tasks)
     metrics["avg_suc_time"] = (total_suc_time /
                                num_solved if num_solved > 0 else float("inf"))
+    total_skeletons_optimized = approach.metrics[
+        "total_num_skeletons_optimized"]
```

Comment:
where is this field set? i don't see it referenced anywhere else in this PR

Reply:
https://github.com/Learning-and-Intelligent-Systems/predicators/pull/298/files#diff-9da13d449b05e590841f4f3c1507a93305d3f0268fe27d36fd4e734aa6032648L44

Reply:
thanks, i forgot that i coded that horribly 🤦‍♂️